src/proto.rs

```diff
@@ -576,22 +639,62 @@ impl RequestRegistrationExtensionsBuilder {
     ///
     /// assert_eq!(extensions.cred_protect, Some(cred_protect));
     /// ```
-    pub fn cred_protect(mut self, cred_protect: RequestCredProtectExtension) -> Self {
+    pub fn cred_protect(mut self, cred_protect: CredProtect) -> Self {
         self.0.cred_protect = Some(cred_protect);
         self
     }
+
+    /// Set the credential blob extension options
+    ///
+    /// # Example
+    /// ```
+    /// # use webauthn_rs::proto::{RequestRegistrationExtensions, CredBlobSet};
+    /// let cred_blob = vec![0xde, 0xad, 0xbe, 0xef];
+    /// let extensions = RequestRegistrationExtensions::builder()
+    ///     .cred_blob(cred_blob.clone())
+    ///     .build();
+    ///
+    /// assert_eq!(extensions.cred_blob, Some(CredBlobSet::from(cred_blob)));
+    /// ```
+    pub fn cred_blob(mut self, cred_blob: Vec<u8>) -> Self {
```

Comment:
I'm wondering if we should length check this, and do Result<Self, Self>, where Err(Self) is the builder but without cred_blob set. 

IIRC there was a 32byte limit?

Reply:
I think implementing authenticators are required at accept at least 32 bytes of data, but technically can accept more

Reply:
This is actually one of the reasons you can get a `False` return on the way back

Reply:
Ahhh because you requested to store something too large. Got it. Okay in that case, this is all good :)